CHANGELOG.md

```diff
@@ -2,9 +2,9 @@
 
 ### Bugfixes
 
-* Fixed various bugs in aggregate methods of Table, TableView and Query for nullable columns.
-  Non-nullable columns might not have been affected. One of the bugs are described here
-  https://github.com/realm/realm-core/issues/2357
+* Fixed various bugs in aggregate methods of Table, TableView and Query for nullable columns
+  (max, min, avg, sum). Non-nullable columns might not have been affected. One of the bugs are
```

Comment:
What did they do wrong? Can we be a bit more specific, so we can write a proper release note in SDKs and inform people know how they might have been affected?